src/plugins/formvalid/formvalid.js

```diff
@@ -374,12 +374,20 @@ var componentName = "wb-frmvld",
 					//Trigger validation on wb-server-error
 					$form.find( ".wb-server-error" ).filter( ":not( :hidden )" ).parent().each( function() {
 						if ( this.attributes.for && this.attributes.for.value.length > 0 ) {
-							$( "form" ).validate().element( $( "[id =" + this.attributes.for.value + "]" ) );
+							$form.validate().element( $( "[id =" + this.attributes.for.value + "]" ) );
 						} else if ( $( this ).find( "input" )[ 0 ] ) {
-							$( "form" ).validate().element( $( this ).find( "input" )[ 0 ] );
+							$form.validate().element( $( this ).find( "input" )[ 0 ] );
 						} else if ( $( this ).next( ".radio, .checkbox" ).children( "label" ).children( "input" )[ 0 ] ) {
-							if ( $( this ).find( $( this ).next( ".radio, .checkbox" ).children( "label" ).children( "input " )[ 0 ].id ) ) {
-								$( "form" ).validate().element( $( this ).next( ".radio, .checkbox" ).children( "label" ).children( "input" )[ 0 ] );
+							if ( $( this ).find( $( this ).next( ".radio, .checkbox" ).children( "label" ).children( "input" )[ 0 ].id ) ) {
+								$form.validate().element( $( this ).next( ".radio, .checkbox" ).children( "label" ).children( "input" )[ 0 ] );
+							}
+						} else if ( $( this ).next( ".radio-inline" ).children( "input" )[ 0 ] ) {
+							if ( $( this ).find( $( this ).next( ".radio-inline" ).children( "input" )[ 0 ].id ) ) {
+								$form.validate().element( $( this ).next( ".radio-inline" ).children( "input" )[ 0 ] );
+							}
+						} else if ( $( this ).next( ".label-inline" ).children( "input" )[ 0 ] ) {
+							if ( $( this ).find( $( this ).next( ".label-inline" ).children( "input" )[ 0 ].id ) ) {
+								$form.validate().element( $( this ).next( ".label-inline" ).children( "input" )[ 0 ] );
 							}
 						}
```

Comment:
Merge those conditions.
```suggestion
						} else if ( $( this ).next( ".radio-inline, .checkbox-inline, .label-inline" ).children( "input" )[ 0 ] ) {
							if ( $( this ).find( $( this ).next( ".radio-inline, .checkbox-inline, .label-inline" ).children( "input" )[ 0 ].id ) ) {
								$form.validate().element( $( this ).next( ".radio-inline, .checkbox-inline, .label-inline" ).children( "input" )[ 0 ] );
							}
						}
```

Reply:
Condition combined.